server/test/user.route.v1.test.ts

```diff
@@ -18,56 +18,107 @@ describe("User Route v1", () => {
     let user1: SocoboUser = new SocoboUser();
```

Comment:
Okay. Could you please refactor the tests? I don't have any clue how to test this the right way. :-(